config.py

```diff
@@ -63,6 +63,13 @@ class Config(object):
                                              RM_SURVEY_SERVICE_HOST,
                                              RM_SURVEY_SERVICE_PORT)
 
+    RM_SAMPLE_SERVICE_HOST = os.getenv('RM_SAMPLE_SERVICE_HOST', 'localhost')
+    RM_SAMPLE_SERVICE_PORT = os.getenv('RM_SAMPLE_SERVICE_PORT', 8125)
```

Comment:
Have these been added to the readme?

Reply:
The readme does not currently mention config variables.

Reply:
I had a chat with @benjefferies about this a while back and the idea was that rather than writing out all the environment variables in the readme we'd just have a link to the config file

Reply:
Has this been made aware to Dave or Mark?

Reply:
i definitely think that we should be listing the the environment variables in the readme (at least the mandatory ones), the readme should give all info required to setup/startup the service

Reply:
@davidmort it looks like the environmental variables documented in the readme is just a duplication of what is in the readme now. Also moves away from one source of the truth.

Would a preferable approach would be that the app works out the box and therefore doesn't documentation to get up and running?

Reply:
@benjefferies I think as it is the application will work out of the box but will try to connect to other services on localhost?
There's a lot of cases currently for developers where they'd need to change that though